.github/workflows/github-external-pr-monitor.yml

```diff
@@ -0,0 +1,18 @@
+name: "GitHub external PRs Monitor"
```

Comment:
I would convert this into a composite action, See https://github.com/MystenLabs/sui-operations/blob/main/.github/actions/slack-message/action.yml as an example.